src/core/gc/config.d

```diff
@@ -12,16 +12,19 @@ import core.internal.parseoptions;
 
 __gshared Config config;
 
+/// UDA for field treated as memory value
+struct MemVal {}
```

Comment:
Thanks. `MemVal` should be declared in parseoptions.d, so there is no circular dependency and it is available for other options, too. The conversion methods could be static members of the struct.

Reply:
> Thanks. MemVal should be declared in parseoptions.d, so there is no circular dependency and it is available for other options, too.

Done

> The conversion methods could be static members of the struct.

I don’t like that. In the future, maybe someone else uses these functions somewhere.